README_BUILD.md

```
@@ -0,0 +1,120 @@
+- `r_type_client` - The game client
+- `r_type_server` - The game server
+
+## Debug Builds
+
+For debug builds, use:
+```bash
+./build.sh debug
+```
+
+Or manually:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)